dev/com.ibm.ws.security_fat.common.tooling/bnd.bnd

```diff
@@ -0,0 +1,50 @@
+#*******************************************************************************
```

Comment:
Will this be used on the commercial liberty repo still?  If so you will probably want to set the publish attribute so a stub project is created over there pointing to it on the open liberty side.

Reply:
Do I need to set this?

`publish.wlp.jar.disabled: false`

Reply:
yes if it is needed both other fats outside of open liberty

Reply:
Updated the bnd.bnd file.